bug: fix move tool

DSCaskey opened the pull request:
This fixes the issue when using the Move tool with the Center point selected when using a language other than English.


- Origin point drop down now correctly translates the Center point item.

![move](https://github.com/FashionFreedom/Seamly2D/assets/31944718/db8471c2-df74-46ab-b76d-3cf5b41f0eae)

- Language can also be changed while Move tool is open in the Properties Editor without throwing the missing point warning().
- Translation files updated as per fix.
- Initialization of Properties Editor moved to separate method to allow the editor to be deleted and reinitialized.
- Fixes tabifying the dock widget.


This fixes issue #1042

DSCaskey committed:
fix move tool

DSCaskey committed:
remove un-needed event() override

ghost commented:
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=1043&src=https%3A%2F%2Fgithub.com%2FFashionFreedom%2FSeamly2D">Click on the image</a> for a new way to code review</b>
</summary>

####

[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/FashionFreedom/Seamly2D/1043/7d3a7add/eb847ca2258c3168011d1feebb31baf43c6bdcc4.svg)](https://app.codesee.io/r/reviews?pr=1043&src=https%3A%2F%2Fgithub.com%2FFashionFreedom%2FSeamly2D)

#### Legend
<img src="https://app.codesee.io/Legend_v3.png" alt="CodeSee Map legend" width="220" height="156">
</details>

csett86 commented on src/libs/vtools/dialogs/tools/dialogmove.cpp:
```diff
@@ -484,7 +486,7 @@ void DialogMove::originChanged(const QString &text)
     VisToolMove *operation = qobject_cast<VisToolMove *>(vis);
     SCASSERT(operation != nullptr)
-    if (text == "Center Point")
+    if (text == tr("Center Point"))
```
One question, because I really dont know: Here its with capital P, while in the vmovetool with lowercase p for point. Does that matter?

DSCaskey replied:
Yes. It should match the properties browser. Although to be honest I don't know if this routine is ever reached. I will make the change.

csett86 commented on src/libs/vtools/tools/drawTools/operation/vtoolmove.cpp:
```diff
@@ -466,13 +466,13 @@ void VToolMove::setFormulaRotation(const VFormula &value)
+    if (m_originPointId == NULL_ID)
     {
-        return VAbstractTool::data.GetGObject(m_originPointId)->name();
+        return tr("Center point");
```
Lowercase p here

DSCaskey replied:
Yes. This is correct.

DSCaskey committed:
Fix text string

Fix the text string in the DialogMoveTool to match the string in the Properties Browser.

DSCaskey commented:
Fixed the text string in DialogMoveTool. Probably should refactor the code at some point to use a global string,. That way only one string would need to be translated, and it would avoid mismatched strings in the 2 classes.

csett86 reviewed (approved):
Looks good to me
